packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -66,4 +67,112 @@ export interface ICheckboxProps extends React.HTMLAttributes<HTMLElement | HTMLI
    * @default 'start'
    */
   boxSide?: 'start' | 'end';
+
+  /**
+   * Theme provided by HOC.
+   */
+  theme?: ITheme;
+
+  /**
+   * Custom styles for this component
+   */
+  styles?: ICheckboxStyles;
 }
+
+export interface ICheckboxStyles {
+  /**
+   * Style for the root element of the checkbox component in the default enabled/unchecked state.
```

Comment:
I was referring more to the comment where you say default enabled/unchecked, because there are no corresponding things here for other states.